.github/workflows/release-channels.yml

```diff
@@ -0,0 +1,52 @@
+name: ZITADEL Release
```

Comment:
Please rename this since this name is already taken

Reply:
corrected